src/studio/src/designer/frontend/packages/schema-editor/src/components/SchemaInspector.tsx

```diff
@@ -437,14 +487,14 @@ const SchemaInspector = ((props: ISchemaInspectorProps) => {
             aria-label='inspector tabs'
           >
             <Tab
-              label={getTranslation('schema_editor.properties', props.language)} {...a11yProps(0)}
+              label={getTranslation('properties', props.language)} {...a11yProps(0)}
             />
             <Tab
-              label={getTranslation('schema_editor.restrictions', props.language)} {...a11yProps(1)}
+              label={getTranslation('restrictions', props.language)} {...a11yProps(1)}
             />
             <Tab
```

Comment:
Could w make a TranslatedTab component so that we can reduce duplications?
we can do the get translation in there